### PR TITLE
DOC-8579 DOC-8580 Docs for Datadog and Cloudwatch metrics integration for Standard clusters

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-8579 DOC-8580

Created Export Metrics Standard page by copying [Export Metrics Dedicated page](https://github.com/cockroachdb/docs/blob/main/src/current/cockroachcloud/export-metrics.md?plain=1) and modifying as follows: (1) replaced site.data.products.dedicated with site.data.products.standard. (2) replaced instances of nodes with regions. (3) removed repetition in intro sentence for Datadog. (4) modified 2 steps for enabling AWS Cloudwatch.

In sidebar, added link to Export Metrics Standard page.

### CODE REVIEW

File: src/current/cockroachcloud/export-metrics-standard.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+## Enable metrics export
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="aws-metrics-export">AWS CloudWatch</button>
+  <button class="filter-button" data-scope="datadog-metrics-export">Datadog</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="aws-metrics-export">
+
+Exporting metrics to AWS CloudWatch is only available on CockroachDB Standard clusters which are hosted on AWS. If your CockroachDB Standard cluster is hosted on GCP or Azure, you can [export metrics to Datadog](export-metrics.html?filters=datadog-metrics-export) instead.
```
Comment (jaylim-crl) [inline src/current/cockroachcloud/export-metrics-standard.md:53]: We don't offer Azure standard clusters today. Not sure if we want to include that. Note that there are a couple of Azure references in the doc.

File: src/current/cockroachcloud/export-metrics-standard.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+{{site.data.alerts.callout_info}}
+Enabling metrics export will send around 250 metrics per region to AWS CloudWatch. Review the [AWS CloudWatch documentation](https://aws.amazon.com/cloudwatch/pricing/) to gauge how this adds to your AWS CloudWatch spend.
```
Comment (jaylim-crl) [inline src/current/cockroachcloud/export-metrics-standard.md:56]: For tenants, there will be 84 metrics per region. We could round it up to 90.

File: src/current/cockroachcloud/export-metrics-standard.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/cloudwatch \
+  --header "Authorization: Bearer {secret_key}"
+~~~
+
+Where:
+
+- `{cluster_id}` is your CockroachDB Standard cluster's cluster ID, which can be found in the URL of your [Cloud Console](https://cockroachlabs.cloud/clusters/) for the specific cluster you wish to configure, resembling `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
```
Comment (jaylim-crl) [inline src/current/cockroachcloud/export-metrics-standard.md:241]: nit: maybe `CockroachDB Standard cluster ID`?

File: src/current/cockroachcloud/export-metrics-standard.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/metricexport/datadog \
+  --header "Authorization: Bearer {secret_key}"
+~~~
+
+Where:
+
+- `{cluster_id}` is your CockroachDB Standard cluster's cluster ID, which can be found in the URL of your [Cloud Console](https://cockroachlabs.cloud/clusters/) for the specific cluster you wish to configure, resembling `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
```
Comment (jaylim-crl) [inline src/current/cockroachcloud/export-metrics-standard.md:259]: nit: maybe `CockroachDB Standard cluster ID`?

**Review by jaylim-crl — APPROVED** [review]
LGTM

File: src/current/cockroachcloud/export-metrics-standard.md (hunk 1 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-standard.md:53]: removed references to Azure.

File: src/current/cockroachcloud/export-metrics-standard.md (hunk 2 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-standard.md:56]: updated from 250 to 90.

File: src/current/cockroachcloud/export-metrics-standard.md (hunk 3 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-standard.md:241]: updated to "cluster ID of your CockroachDB Standard cluster"

File: src/current/cockroachcloud/export-metrics-standard.md (hunk 4 above)
Comment (florence-crl) [reply src/current/cockroachcloud/export-metrics-standard.md:259]: updated to "cluster ID of your CockroachDB Standard cluster"

**Review by florence-crl — COMMENTED** [review]
TFTR

**Review by rmloveland — APPROVED** [review]
LGTM

Too bad there's a lot of duplication here but over time maybe we can do this with conditional logic once the whole Standard etc. situation shakes out more fully

**Pull request merged**